### PR TITLE
1889 Allow multiple respondents 

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2019-06-19 16 51 47

### CODE REVIEW

File: shared/src/business/entities/Case.js [hunk 1]
```diff
@@ -250,10 +250,11 @@ joiValidationDecorator(
       .max('now')
       .optional()
       .allow(null),
-    respondent: joi
-      .object()
-      .allow(null)
-      .optional(),
+    respondents: joi.array().items(
+      joi.object().keys({
+        respondentId: joi.string().uuid(uuidVersions),
```
Comment (rachaelparris) [inline shared/src/business/entities/Case.js]: Should this be `userId`?

File: web-client/src/views/CaseDetail/PartyInformation.jsx [hunk 2]
```diff
@@ -71,24 +71,27 @@ export const PartyInformation = connect(
...
-                </address>
-              </React.Fragment>
-            )}
-          </div>
+          {caseDetail.respondents &&
+            caseDetail.respondents.map((respondent, index) => (
+              <div className="tablet:grid-col-2" key={index}>
+                {index === 0 && (
+                  <p className="label" id={'respondent-label'}>
+                    Petitioner Counsel
```
Comment (rachaelparris) [inline web-client/src/views/CaseDetail/PartyInformation.jsx]: `Respondent Information`?

Comment (wbyoko) [reply web-client/src/views/CaseDetail/PartyInformation.jsx]: yup 👍

File: shared/src/business/entities/Case.js (hunk 1 above)
Comment (wbyoko) [reply shared/src/business/entities/Case.js]: not really. I'll remove that definition.

**Review by codyseibert — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**